src/i18n/resources/en/admin.json

```diff
@@ -1,7 +1,11 @@
 {
   "admin": "Admin",
+  "country_few": "{{count}} Countries",
+  "country_many": "{{count}} Countries",
   "country_one": "{{count}} Country",
   "country_other": "{{count}} Countries",
+  "country_two": "{{count}} Countries",
+  "country_zero": "{{count}} Countries",
```

Comment:
any chance to use `$t(admin.country_other, {{count}})` (and for all others).
This will avoid replicating the same word. what do you think? can you please verify this will work ?